shared/common-adapters/avatar.tsx

```diff
@@ -101,6 +102,8 @@ const ConnectedAvatar = Container.connect(
     _followsYou: ownProps.showFollowingStatus ? state.config.followers.has(ownProps.username || '') : false,
     _httpSrvAddress: state.config.httpSrvAddress,
     _httpSrvToken: state.config.httpSrvToken,
+    blocked:
```

Comment:
Some changes need to be made to `sync-avatar-props.desktop` to have this work in remote windows, the menubar throws an error on this PR.

Reply:
I just changed this so it doesn't need the state, since we shouldn't be showing any blocked users in the widget anyway (via inbox and TLF updates).